metadata-ingestion/src/datahub/ingestion/source/snowflake/snowflake_schema.py

```diff
@@ -449,8 +465,53 @@ def get_views_for_database(self, db_name: str) -> Dict[str, List[SnowflakeView]]
                 )
                 view_pagination_marker = view_name
 
+        # Because this is in a cached function, this will only log once per database.
+        view_counts = {schema_name: len(views[schema_name]) for schema_name in views}
+        logger.info(
+            f"Finished fetching views in {db_name}; counts by schema {view_counts}"
+        )
+        return views
+
+    @classmethod
+    def _map_view(cls, row: Dict[str, Any]) -> Tuple[str, SnowflakeView]:
+        schema_name = row["VIEW_SCHEMA"]
+        return schema_name, SnowflakeView(
+            name=row["VIEW_NAME"],
+            created=row["CREATED"],
+            comment=row["COMMENT"],
+            view_definition=row["VIEW_DEFINITION"],
```

Comment:
Note to self: this needs to be populated via `show views` in cases where the role is not the owner of the view